### PR TITLE
chore(storage): rename storage secret

### PR DESCRIPTION
**Pull request by mwangggg**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits): `git commit -S -m "YOUR_COMMIT_MESSAGE"`
_______________________________________________

Fixes: #832

### CODE REVIEW

File: bundle/tests/scorecard/config.yaml [hunk 1]
```diff
@@ -120,13 +120,23 @@ stages:
...
     - cryostat-report
-    image: quay.io/cryostat/cryostat-operator-scorecard:3.0.0-20240511064726
+    image: quay.io/cryostat/cryostat-operator-scorecard:3.0.0-20240527193829
     labels:
       suite: cryostat
       test: cryostat-report
     storage:
       spec:
         mountPath: {}
+  - entrypoint:
```
Comment (tthvo) [inline bundle/tests/scorecard/config.yaml:130]: Maybe not include these scorecard updates?

Comment (andrewazores) [reply bundle/tests/scorecard/config.yaml:130]: ^ https://github.com/cryostatio/cryostat-operator/pull/815#issuecomment-2133983253

Comment (tthvo) [reply bundle/tests/scorecard/config.yaml:130]: oh right haha :))

**Review by tthvo — COMMENTED** [review]
Looks good to me!

**Review by andrewazores — APPROVED** [review]

**Pull request merged**